report/4.Design/RTS.tex

```diff
@@ -6,7 +6,73 @@ \subsection{Scheduling}\label{Design:Scheduling}
 When choosing a scheduling method for the system, multiple methods were considered. 
 Both fixed priority scheduling and cyclic executive were considered for the system as both of them would be good at handling the tasks of the system with their deadlines.
 
-Cyclic executive was chosen due the the fast execution time of the tasks, and that preemption and context switching would exceed the time for simply completing the previous task. 
-It was reasoned that since all tasks should run regularly, and execute fast enough to run within the major cycle the cyclic execution method should be perfect. 
+Due to the fast execution time of the tasks, and that preemption and context switching would exceed the time for simply completing the previous task, it was at first deemed obvious to use the cyclic executive model as all tasks should run regularly, and that they execute fast enough to run within the major cycle the cyclic execution method should be perfect. 
```

Comment:
Lang sætning.. Consider rephrasing.

Reply:
Især sidste den bliver meget svær at forstå  fra "executive model as..." og til slut. Der mangler som minimum komma "major cycle, the cyclic execution"